traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -236,11 +236,63 @@ <h4>New Term Information</h4>
       </div>
     </section>
     {% endif %}
+
+    <section class="section">
+      <h4>TRAIT HISTORY</h4>
+
+      {% if not user.is_anonymous %}
+      <div class="post-comment__container">
+        <div class="post-comment__header">
+          <span uk-icon="user"></span>
+          <span>Comment as {{user.get_full_name}}</span>
+          <button class="uk-button button button--primary post-comment__submit" onclick="commentButtonClicked()">Submit</button>
+        </div>
+
+        <div class="post-comment__body">
+          <textarea 
+          id="commentBody" 
+          class="uk-textarea post-comment__textarea" 
+          placeholder="Leave a commment..."></textarea>
+        </div>
+      </div>
+      {% endif%}
```

Comment:
![image](https://user-images.githubusercontent.com/10669118/89111432-8fe5e500-d44d-11ea-9a4c-50212229df35.png)

For the mapping change entry, I feel this could be changed to (note the em dash and making the CURIE part clickable):

**Kirill Tsukanov** proposed a mapping on **August 1st 2020, 23:08**  
**[Orphanet:101016](http://www.orpha.net/ORDO/Orphanet_101016) — Romano-Ward syndrome**